docs/zh-CN/administrator-guide/http-actions/fe/query-detail-action.md

```diff
@@ -0,0 +1,116 @@
+---
+{
+    "title": "Query Detail Action",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Query Detail Action
+
+## Request
+
+`GET /api/query_detail`
+
+## Description
+
+用于获取指定时间点之后的所有查询的信息
```

Comment:
How many can be returned at most?

Reply:
All of them.
This can be improved later. Now we just keep the origin behavior